linode_api4/objects/monitor.py

```diff
@@ -0,0 +1,82 @@
+from linode_api4.objects import Base, Property
+
+
+class Dashboard(Base):
+    """
+    List dashboards: https://techdocs.akamai.com/linode-api/get-dashboards-all
+    """
+
+    api_endpoint = "/monitor/dashboards/{id}"
+    properties = {
+        "id": Property(identifier=True),
+        "created": Property(is_datetime=True),
+        "label": Property(),
+        "service_type": Property(),
+        "type": Property(),
+        "widgets": Property(mutable=True),
+        "updated": Property(is_datetime=True),
+    }
+
+
+class DashboardByService(Base):
+    """
+    Get a dashboard: https://techdocs.akamai.com/linode-api/reference/get-dashboards
+    """
+
+    properties = {
+        "id": Property(identifier=True),
+        "created": Property(is_datetime=True),
+        "label": Property(),
+        "service_type": Property(),
+        "type": Property(),
+        "widgets": Property(mutable=True),
+        "updated": Property(is_datetime=True),
+    }
+
+
+class MonitorServiceSupported(Base):
+
+    api_endpoint = "/monitor/services/"
+    id_attribute = "service_type"
+    properties = {
+        "service_type": Property(),
```

Comment:
You should add the identifier label to the property
```suggestion
        "service_type": Property(identifier=True),
```
I want to double check, is there an endpoint to get a single supported service?